### DISCUSSION TITLE
Yi Dome R30GB chinese version

### DISCUSSION THREAD

**Discussion by Delfistyaosani** [roleoroleo/yi-hack-Allwinner-v2#231] [Q&A]
Okay gentlemen, my journey began with a fk up: bought a pair of dome cams from taobao because it was cheaper.
When trying to use it with their app unmodified - it spoke in chinese all way, except when it was about to pair I got the pleasant "this device can only be used within china" message.
I did try to apply the respective hack with unpacked TAR and now I got telnet working but nothing else.
The cloud tries to pair but times out.
The thing still speaks in chinese.
No web server no config page nothing.
I don't even see find command in the telnet.
No systemctl enable ssh. Port 23 is literally the only one open.
Any ideas? Please help.

**Comment by roleoroleo** [comment]
What is your fw revision?

**Reply by Delfistyaosani** [reply]
I'm kinda noob in this topic, how do I get details other than the R30GB which is written on the device?
I did backup the original fw before meddling with it, can post the dump if it helps.

**Reply by roleoroleo** [reply]
Read it inside the original app.
